test/k8sT/Nightly.go

```diff
@@ -433,8 +433,9 @@ var _ = Describe("NightlyExamples", func() {
 
 					err := kubectl.CiliumEndpointWaitReady()
 					Expect(err).To(BeNil(), "Endpoints are not ready after timeout")
-
-					cleanupCallback = ValidateCiliumUpgrades(kubectl)
+					var test func()
+					test, cleanupCallback = ValidateCiliumUpgrades(kubectl)
+					test()
```

Comment:
I don't understand, what is "test"? This is a really vague name, can you give it a better name?

Reply:
give me a name, I thought about it and I can't get better name :-( it's a test

Reply:
What does the test do? Seems like it asserts that the upgrade has occurred correctly? Would something like `assertUpgradeSuccessful` be accurate?